builtin/providers/vsphere/resource_vsphere_virtual_machine.go

```diff
@@ -418,6 +436,19 @@ func resourceVSphereVirtualMachineRead(d *schema.ResourceData, meta interface{})
 	d.Set("datacenter", dc)
 	d.Set("memory", mvm.Summary.Config.MemorySizeMB)
 	d.Set("cpu", mvm.Summary.Config.NumCpu)
+
+	if mvm.Config != nil && mvm.Config.ExtraConfig != nil && len(mvm.Config.ExtraConfig) > 0 {
+		//TODO: can only set specific custom value, not everything
+		//Would need the config here
+		//custom_configs := make(map[string]types.AnyType)
+		for _, v := range mvm.Config.ExtraConfig {
+			value := v.GetOptionValue()
+			//custom_configs[value.Key] = value.Value
+			log.Printf("[DEBUG] custom configs %s,%s",value.Key, value.Value)
+		}
+		//d.Set("custom_configuration_parameters", custom_configs)
+	}
```

Comment:
we can simply remove this section.  I was attempting to pull back the custom config values that are being set, but the list is a lot longer that provided.